SPECS-EXTENDED/javapackages-tools-meta/javapackages-tools-meta.signatures.json

```diff
@@ -1,5 +1,3 @@
 {
- "Signatures": {
-  "javapackages-tools-meta-5.3.0.tar.gz": "73803877c26b3e6c0440b2e1e4d78a5fadf63782a62374df400b8badb1de3046"
- }
+ "Signatures": {}
```

Comment:
I'm personally fine with keeping this stub, but we could also just remove this file for consistency (`filesystem` is one example from our core specs)

Reply:
agree with Thomas.  File removal is preferred.

Reply:
Sure, `removal` is better and consistent.